app/admin/settings/Amounts.scala

```diff
@@ -0,0 +1,22 @@
+package admin.settings
+
+import io.circe.{Decoder, Encoder}
+
+case class Amount(value: String, isDefault: Option[Boolean])
```

Comment:
The value is a string on the client, so for simplicity I've kept it that was here as well.
This is a bit wrong, but I'm not sure it's worth changing